.github/workflows/cpu-integration-tests.yaml

```diff
@@ -0,0 +1,35 @@
+name: CPU Integration Tests
+on: [push]
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  integration-tests:
+    runs-on:
+      labels: ubuntu-22.04-64core
+    steps:
+      - uses: actions/checkout@v4
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and export to Docker
```

Comment:
We first build the docker image so we don't have to build it thrice when spinning up the docker compose